api/sixtyfps-rs/migration.md

```diff
@@ -56,6 +56,22 @@ fn model_tracker(&self) -> &dyn ModelTracker {
 }
 ```
 
+#### `Option<SharedModel>` replaces `ModelHandle`
+
+`Option<SharedModel>` replaces `ModelHandle`.
+
+Old code:
+
+```rust
```

Comment:
I don't think you need to always put code sample for such trivial things like that.
Anyway, the ```rust must compile as it is going to be a doctest, so you need to add  `ignore` 
But i realize that this file is not included in the rust doc yet.